pkg/configs/api/api.go

```diff
@@ -108,7 +108,7 @@ func (a *API) RegisterRoutes(r *mux.Router) {
 
 // getConfig returns the request configuration.
 func (a *API) getConfig(w http.ResponseWriter, r *http.Request) {
-	userID, _, err := user.ExtractOrgIDFromHTTPRequest(r)
+	userID, _, err := tenant.ExtractTenantIDFromHTTPRequest(tenant.DefaultResolver(), r)
```

Comment:
**suggestion(if-minor):** similar to the suggestion above, what if the default resolver is assumed and then either add a separate `ExtractOrgIDFromHTTPRequestWithResolver` function now or later when it is needed.

Reply:
Done that, I haven't added a method with resolver argument, as we can easily do that once it becomes important